src/module/behaviors.js

```diff
@@ -0,0 +1,230 @@
+/* eslint-disable no-useless-concat */
+export const gameCardsElement = document.querySelector('#game-cards');
+export const options = {
+  method: 'GET',
+  headers: {
+    'X-RapidAPI-Key': '16a429eb7fmshb30918cd461ee6dp1673e1jsn62d7843ea7e3',
+    'X-RapidAPI-Host': 'free-to-play-games-database.p.rapidapi.com',
+  },
+};
+
+export const invAPI = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/Kksh8kGPsQh7xrD8jbYD';
+
+export const likeGame = async (id) => {
+  const API_URL = `${invAPI}/likes`;
+  return fetch(API_URL, {
+    method: 'POST',
+    body: JSON.stringify({
+      item_id: id,
+    }),
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  });
+};
+
+export const getLikes = async () => {
+  const API_URL = `${invAPI}/likes`;
+  const res = await fetch(API_URL);
+  const data = await res.json();
+  const objetc1 = {};
+  data.forEach((element) => {
+    objetc1[element.item_id] = element.likes !== undefined ? element.likes : 0;
+  });
+  return objetc1;
+};
+
+export const getComments = async (id) => {
+  const API_URL = `${invAPI}/comments?item_id=${id}`;
+  const response = await fetch(API_URL);
+  return response.json();
+};
+
+export const addComment = async (gameId, name, comment) => {
+  const API_URL = `${invAPI}/comments`;
+  const response = await fetch(API_URL, {
+    method: 'POST',
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+    body: JSON.stringify({
+      item_id: gameId,
+      username: name,
+      comment,
+    }),
+  });
+  if (response.ok) {
+    // Refresh comments
+    getComments(gameId);
+    return response;
+  }
+  throw new Error(`Request failed with status ${response.status}`);
+};
+
+export const fetchData = () => new Promise((resolve, reject) => {
+  fetch(
+    'https://free-to-play-games-database.p.rapidapi.com/api/filter?tag=3d.mmorpg.fantasy.pvp&platform=pc',
+    options,
+  )
+    .then((response) => response.json())
+    .then((data) => {
+      resolve(data);
+    })
+    .catch((err) => reject(err));
+});
+export const likes = {};
+
+export const createGameCardHTML = (game, index) => `
+  <div class="game-card">
+    <img class="game-img" src="${game.thumbnail}" alt="${game.title}" />
+    <h2 class="game-title">${game.title}</h2>
+    <p class="game-desc">${game.short_description.slice(0, 200)}${
+  game.short_description.length > 200 ? '...' : ''
+}</p>
+    <div class="game-actions">
+      <a class="game-link" href="${game.game_url}" target="_blank">Play Now!</a>
```

Comment:
- [ ] Please kindly remove the `Play now` button as there is no provision for it in the design.